### ISSUE TITLE
No progress indicators

### ISSUE THREAD

**Issue by halr9000** [Animator-Anon/animator_extension#3]
Cool project, and so far finding it easier to use than deforum. It would be nice to have some better progress indicators, is that something you are working on? 

After hitting Process, all I see is "Loading". If I switch tabs and come back, it will sometimes, but not always load images. If it don't, it will not. 

Some UI suggestions to go along with progress:

- instead of process/stop, be consistent with other tabs and use generate/interrupt
- apply same styles as other tabs to the process/generate button (default is white on orange), and switch the style during processing so that it's obvious something is happening

**Comment by Animator-Anon** [comment]
Yeah I have thought about it but haven't found an easy solution yet. What WubUI does seems a bit too complicated. I tend to watch the commandline window for some indication. And I agree it should be consistent. 
Maybe ChatGPT3 can figure it out for me...
